### DISCUSSION TITLE
Understanding installed skills list

### DISCUSSION THREAD

**Discussion by atd** [OpenVoiceOS/ovos-buildroot#133] [Q&A] [answered]
I just installed the buildroot image and I am trying to understand and locate the current skills list, so I can debug and fix issues related to Spanish

From the `cli_login.sh`, it seems I should use `msm`. However, I cannot current installed skills there. For instance, I can see the weather working, but using `msm list | grep weather` I can only see `mycroft-weather`, which is not installed

```
$ msm info mycroft-weather
INFO - building SkillEntry objects for all skills
INFO - Best match (1.0): mycroft-weather by mycroftai
Name: mycroft-weather
Author: mycroftai
Url: https://github.com/mycroftai/skill-weather
Path: Not installed
```

I cannot see them in `.local/share/mycroft/skills.json` which seems to be the list of user-installed skills

I cannot find other `skills.json` files

Somebody could shed some light on how this is working in OVOS?

**Comment by NeonDaniel** [comment, marked as answer]
Several skills in OVOS are installed as python packages, rather than cloned directories (what MSM looks for). I think the easiest way to get a list of skills is to query the skills service which you can do with the `mana` CLI tool:

```
pip install neon-mana-utils
mana get-skills-list
```

**Comment by j1nx** [comment]
Or within the ovos-cli-client you type the command

```
:skills
```
That will give you the loaded skills. Yellow ones are all loaded fine, the red ones failed to load.

However the will not tell where they are installed (good idea actually @JarbasAl don't you agree?)
